File: 2019.09/json/java/Main.java
import com.google.gson.Gson;

class Hoge {
  public float foo;
  public double bar;
}

public class Main {
  public static void main(String[] args) {
    Hoge hoge = new Hoge();
    hoge.foo = 0xa0000000L;
    hoge.bar = 0xa0000000L;
    Gson gson = new Gson();
    System.out.println(gson.toJson(hoge));
  }
}

File: 2019.09/json/go/main.go
package main

import (
	"encoding/json"
	"fmt"
)

type Hoge struct {
	Foo float32
	Bar float64
}

func main() {
	hoge := Hoge{Foo: 0xa0000000, Bar: 0xa0000000}
	j, _ := json.Marshal(hoge)
	fmt.Println(string(j))
}
